README.adoc

```diff
@@ -232,38 +240,47 @@ fun `WHEN putting file on sftp THEN file is read and moved to a correct director
           "move-to-path/test.csv",
         )
 
-        // some other testing you need to do
+        // other testing you need to do on the ingested csv data
+        ...
     }
 }
 ----
 
-=== Absolute vs relative paths
-
-The SFTP Server Stub supports both absolute (`/path/to/file`) and relative (`path/to/file`) paths.
-Quite commonly SFTP servers present client with a limited view of the filesystem, where root of what's available to the user is presented as `/` and that is the initial directory where user starts after login.
-In such a case, absolute and relative paths will behave effectively the same and point to the same place in the filesystem.
+In the above example, we've created a helper extension function called `assertSourceFileProcessingSucceeded()` to verify that the source SFTP has moved the file from the `source` folder to the `moveTo` folder.
+The helper uses `org.awaitility:awaitility` and looks like the following code:
 
-If that's not your case and your server presents you with let's say `/home/myuser` after logging in, then you should set the path when initializing the context.
+[source,kotlin]
 ----
-withSftpServer(initialDirectory = "/home/myuser") {
-  // ...
+private fun SftpServerTestContext.assertSourceFileProcessingSucceeded(
```

Comment:
I suggest we rewrite the code example for brevity and clarity. 
The `logAsserter.awaitEvent`statement can be removed. Its not needed for the example test to verify that the file was successfully moved. Also I suggest we inline the code from this `assertSourceFileProcessingSucceeded` as we are only doing one check, and this function takes up a lot of screen estate and adds neew concepts like exkstention functions, that makes the example less to the point.

Also the `addFlowTestConfig` can have the resource definition removed since a resource is typically not needed in a file-to-file integration. In short, let's see if we can make the example shorter and more focused on stub usage.